app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -76,9 +79,12 @@ def update_in_provider(params)
     with_provider_object do |provider_object|
       provider_object.update_attributes!(params)
     end
+
     if respond_to?(:refresh_in_provider)
```

Comment:
What do you think if we make Line79-81 as a class method, e.g. `refresh_and_notify`?  And Line 15-18 can also call this?

Reply:
👍  This is better than extracting just a one-liner.

It was not as straightforward